test/flows/check_uk_visa_flow_test.rb

```diff
@@ -608,19 +610,34 @@ class CheckUkVisaFlowTest < ActiveSupport::TestCase
   context "outcome: outcome_no_visa_needed" do
     setup do
       testing_node :outcome_no_visa_needed
-      add_responses what_passport_do_you_have?: @eea_country,
-                    purpose_of_visit?: "transit"
+      add_responses purpose_of_visit?: "transit"
     end
 
     should "render a suggestion of evidence for a further journey" do
-      add_responses travelling_to_cta?: "somewhere_else"
+      add_responses what_passport_do_you_have?: @eea_country,
+                    travelling_to_cta?: "somewhere_else"
       assert_rendered_outcome text: "you should bring evidence of your onward journey"
     end
 
     should "render a suggestion of a visa for a further journey to Ireland" do
-      add_responses travelling_to_cta?: "republic_of_ireland"
+      add_responses what_passport_do_you_have?: @eea_country,
+                    travelling_to_cta?: "republic_of_ireland"
       assert_rendered_outcome text: "You may want to apply for a visa"
     end
+
+    should "render specific guidance for Electronic Travel Authorisation" do
+      add_responses what_passport_do_you_have?: @electronic_travel_authorisation_country,
+                    travelling_to_cta?: "somewhere_else",
+                    passing_through_uk_border_control?: "no"
+      assert_rendered_outcome text: "If you’re travelling after 15 November 2023, you’ll need to apply for an electronic travel authorisation (ETA) instead of an electronic visa waiver. You’ll be able to apply for an ETA from 25 October 2023."
```

Comment:
Can we extract the text out to a variable in this file?

Reply:
Yes, I have done this now